app/views/optIn/OptInCompletedView.scala.html

```diff
@@ -96,30 +96,48 @@ <h2 id="optin-completed-view-heading" class="govuk-heading-m" >@messages("optin.
 
 @futureReporitng = {
     <div id="optin-completed-view" class="govuk-!-margin-bottom-7  govuk-!-margin-top-8">
-        <h2 class="govuk-heading-m" >@messages("optin.completedOptIn.futureReporting")</h2>
-        <p id="optin-completed-view-p5" class="govuk-body">
-           @if(model.isCurrentYear) {
-               @messages("optin.completedOptIn.cy.futureReporting.desc1", model.startYear, model.endYear)
-           } else {
-               @messages("optin.completedOptIn.ny.futureReporting.desc1", model.startYear, model.endYear)
-           }
-        </p>
-        <ul class="govuk-list govuk-list--bullet">
-            <li>@messages("optin.completedOptIn.futureReporting.p1")</li>
-            <li>@messages("optin.completedOptIn.futureReporting.p2")</li>
-        </ul>
+        @if(!model.annualWithFollowingYearMandated) {
+            <h2 class="govuk-heading-m" >@messages("optin.completedOptIn.futureReporting")</h2>
+            <p id="optin-completed-view-p5" class="govuk-body">
+            @if(model.isCurrentYear) {
+                @messages("optin.completedOptIn.cy.futureReporting.desc1", model.startYear, model.endYear)
+            } else {
+                @messages("optin.completedOptIn.ny.futureReporting.desc1", model.startYear, model.endYear)
+            }
+            </p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>@messages("optin.completedOptIn.futureReporting.p1")</li>
+                <li>@messages("optin.completedOptIn.futureReporting.p2")</li>
+            </ul>
+        } else {
+            <h2 class="govuk-heading-m" >@messages("optin.completedOptIn.nymandated.futureReporting")</h2>
+            <p id="optin-completed-view-p5" class="govuk-body">
+                @messages("optin.completedOptIn.cy.futureReporting.desc1", model.startYear, model.endYear)
```

Comment:
not sure if I've gotten them the wrong way round now. But think this key needs to be 
```
optin.completedOptIn.nymandated.futureReporting.desc1
```

to change the text to be 
"You have just chosen to voluntarily report quarterly from the {0} to {1} tax year."

when it is CY = Annual CY+1 = Mandated, to match the ticket.